pkg/proxy/winkernel/proxier.go

```diff
@@ -233,7 +206,8 @@ type endpointsInfo struct {
 	hnsID           string
 	refCount        *uint16
 	providerAddress string
-	hns             HostNetworkService
+	hns             IHCNUtils
```

Comment:
Why to have IHCNUtils  interface when HostNetworkingService can serve the purpose.  Both IHCNUtils and HostNetworkService are interfaces and have same functions. And HostNetworkService is more sensible and standard than IHCNUtils.